airflow/api_connexion/endpoints/task_endpoint.py

```diff
@@ -49,11 +51,21 @@ def get_task(dag_id, task_id):
         (permissions.ACTION_CAN_READ, permissions.RESOURCE_TASK_INSTANCE),
     ]
 )
-def get_tasks(dag_id):
+def get_tasks(dag_id, order_by='task_id'):
     """Get tasks for DAG"""
     dag: DAG = current_app.dag_bag.get_dag(dag_id)
     if not dag:
         raise NotFound("DAG not found")
     tasks = dag.tasks
+    if '-' in order_by:
+        try:
+            tasks = sorted(tasks, key=attrgetter(order_by.strip('-')), reverse=True)
+        except AttributeError as err:
+            raise BadRequest(detail=str(err))
+    else:
+        try:
+            tasks = sorted(tasks, key=attrgetter(order_by))
+        except AttributeError as err:
+            raise BadRequest(detail=str(err))
```

Comment:
```suggestion
    try:
        tasks = sorted(tasks, key=attrgetter(order_by.strip('-')), reverse='-' in order_by)
    except AttributeError as err:
        raise BadRequest(detail=str(err))
```

Reply:
Maybe `order_by.lstrip('-')` and `reverse=(orderby[0] == '-')` instead?